roles/kubernetes/control-plane/templates/k8s-certs-renew.sh.j2

```diff
@@ -1,7 +1,30 @@
 #!/bin/bash
 
-echo "## Expiration before renewal ##"
-{{ bin_dir }}/kubeadm certs check-expiration
+echo "## Check Expiration before renewal ##"
+
+do_renew=1 # default: always do renewal
```

Comment:
The boolean with default IMO confuses the control flow.
Can you instead simply act immediately after the checks ?

Reply:
ok, remove it.